webapp/src/main/scala/webapp/Main.scala

```diff
@@ -1,20 +1,38 @@
 package webapp
 
+import cats.effect.SyncIO
+import colibri.Subject
 import outwatch._
 import outwatch.dsl._
-import cats.effect.SyncIO
+import webapp.Main.StyledComponents.{basicButton, basicTextInput}
 
-import colibri.Subject
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSImport
 
 // Outwatch documentation:
 // https://outwatch.github.io/docs/readme.html
 
+@js.native
+@JSImport("../../../../src/main/css/index.css", JSImport.Namespace)
+object Css         extends js.Object
+//
+@js.native
+@JSImport("../../../../src/main/css/tailwind.css", JSImport.Namespace)
+object TailwindCss extends js.Object
```

Comment:
Can we move these into another file? Just because they take up so much space just for importing. Maybe it would also be possible to register css files via webpack config.

Reply:
Good idea. Looks cleaner that way. Don't know too much about webpack though. Could be css-loader might to the trick but I don't want to go down that rabbit-hole. Maybe it's good enough for now - 80/20, low-hanging fruit, quick-win, wissenscho ;-)